components/resume/skills/skills.scss

```diff
@@ -1,6 +1,12 @@
 .resume__skills {
     display: flex;
-    justify-content: space-between;
+    justify-content: space-around;
```

Comment:
These are still a bit wonky, but much better than overflowing off screen on lower resolutions.

![image](https://user-images.githubusercontent.com/667983/165003209-fd1f4910-ce72-4ebe-8e1c-b5b516c77c46.png)

Though there are resolutions where this looks odd
![image](https://user-images.githubusercontent.com/667983/165003227-a7843f06-fb61-4cf5-bf1d-e7a94e9d34e6.png)

Maybe there's a flex wrap related style which realizes the content is on multiple lines now and distributes the spans between both.